git/git_bin.go

```diff
@@ -2,26 +2,38 @@ package git
 
 import (
 	"path/filepath"
+	"sync"
 
 	"github.com/cli/safeexec"
 )
 
+var (
+	gitBinPath string
+	gitBinErr  error
```

Comment:
Looks as if the linter [is really unhappy](https://github.com/github/git-sizer/actions/runs/7198882781/job/19609303857?pr=117#step:5:23) with this:

```
 Error: the variable name `gitBinErr` should conform to the `errXxx` format (errname)
Error: Process completed with exit code 1.
```

But I see that you already addressed this via https://github.com/github/git-sizer/pull/117/commits/a5da5806a959754a8ef47161f45153c9b5fdf2db. How about squashing that commit into this here, seeing as the previous tip commit needs to be dropped from this PR anyway?

Reply:
Done :heavy_check_mark: